string io1_status
uint16 IO_1
string io2_status
uint16 IO_2
uint16 mag
int16 mag_x
int16 mag_y
int16 mag_z